golden_marshmallows/schema.py

```diff
@@ -152,6 +146,10 @@ def __init__(self, sqlalchemy_cls, nested_map={}, new_obj=False,
                 deserialize to new objects; basically just skips adding
                 any field named 'id'
         """
+        nested_map = nested_map if nested_map is not None else {}
+
+        kwargs['unknown'] = unknown
```

Comment:
This flips out the default but should allow you to raise if you want it configured that way.